recipe-client-addon/test/browser/browser_AddonStudies.js

```diff
@@ -218,7 +226,7 @@ decorate_task(
   AddonStudies.withStudies(),
   async function testStopNoStudy() {
     await Assert.rejects(
-      AddonStudies.stop("does-not-exist"),
+      AddonStudies.stop("does-not-exist", "test"),
```

Comment:
Out of curiosity, what breaks if you don't provide a reason here? I thought there was a default value for the argument?

Reply:
To be clear, it makes sense to want to test the propagation of the reason, but that doesn't happen here - you've just updated all the existing consumers and I'm not entirely sure why.

Reply:
To help with my development, I had made all the functions that take `reason` require it, and throw if it was not passed. Then I made all the tests pass, including new tests that included reason propagation. After I got it working, I removed the error throwing and provided a default value to make this code less likely to throw errors.

Thinking about it more, I should have either left the error reporting in, or gone through and removed the no-op reasons in the tests. Do you have opinions about which would be better?

Reply:
I don't really feel all that strongly either way. I think not throwing (and so removing the no-op reasons) might be a slightly safer bet given the few cases in the last 6 months or so where we had issues on startup. It might be nice to have a test that ensures that we end up with the default "unknown" value when no reason is passed.